builds/misc/images-release.yaml

```diff
@@ -0,0 +1,601 @@
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                },
+                {
+                    "keyCode": "CP-230012",
+                    "operationSetCode": "SigntoolVerify",
+                    "parameters": [
+                    {
+                        "parameterName": "VerifyAll",
+                        "parameterValue": "/all"
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                }
+              ]
+          signConfigType: inlineSignParams
+      - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+        displayName: "Functions Binding Code Sign"
+        inputs: 
+          ConnectedServiceName: "Azure IoT Edge Code Sign 2"
+          FolderPath: $(Build.BinariesDirectory)/publish/Microsoft.Azure.WebJobs.Extensions.EdgeHub
+          Pattern: Microsoft.Azure.WebJobs.Extensions*.dll
+          SessionTimeout: 20
+          inlineOperation: |
+              [
+                {
+                    "keyCode": "CP-230012",
+                    "operationSetCode": "SigntoolSign",
+                    "parameters": [
+                    {
+                        "parameterName": "OpusName",
+                        "parameterValue": "Microsoft"
+                    },
+                    {
+                        "parameterName": "OpusInfo",
+                        "parameterValue": "http://www.microsoft.com"
+                    },
+                    {
+                        "parameterName": "Append",
+                        "parameterValue": "/as"
+                    },
+                    {
+                        "parameterName": "FileDigest",
+                        "parameterValue": "/fd \"SHA256\""
+                    },
+                    {
+                        "parameterName": "PageHash",
+                        "parameterValue": "/NPH"
+                    },
+                    {
+                        "parameterName": "TimeStamp",
+                        "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                },
+                {
+                    "keyCode": "CP-230012",
+                    "operationSetCode": "SigntoolVerify",
+                    "parameters": [
+                    {
+                        "parameterName": "VerifyAll",
+                        "parameterValue": "/all"
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                }
+              ]
+          signConfigType: inlineSignParams
+      - template: ../templates/install-dotnet3.yaml
```

Comment:
Again, I think you don't need this step since dotnet 3.1 is already installed on MS hosted Ubuntu 16.04 agent, https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1604-README.md.

Reply:
When I tested this for linux, we need to install the version we want to use next. We can't install both from the beginning.

Reply:
this looks weird to me, what is the exception in Functions binding nuget package step if we don't install dotnet 3.1 here?

Reply:
I forget, but I can rerun and check

Reply:
Take a look at these, https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#project-using-entity-framework-has-stopped-working-on-hosted-agents and https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x.

Reply:
Here is the error:
```
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '2.0.0' was not found.
  - The following frameworks were found:
      3.1.4 at [/opt/hostedtoolcache/dotnet/shared/Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.
```

I am going to revert my latest commit to fix it

Reply:
@philipktlin just saw your comment. For the first link, we use publish scripts and not the dotnet cli, so everything is fine here. For the second link, if we run a command like below, then we can toggle between sdk versions without reinstalling:
```
dotnet new globaljson --sdk-version 3.0.100
```

Do you agree?

Reply:
I think so, but you need to try it out and create a global.json on the root folder of the drive.